daemon/cluster/networks.go

```diff
@@ -311,7 +311,11 @@ func (c *Cluster) populateNetworkID(ctx context.Context, client swarmapi.Control
 			return err
 		}
 	setid:
-		networks[i].Target = apiNetwork.ID
+		if runconfig.IsPreDefinedNetwork(n.Target) {
```

Comment:
This seems to only be reached when `getNetwork` succeeds. Will it succeed, now that swarm is not creating network objects for predefined networks?